### PR TITLE
Btax output tooltips

### PR DESCRIPTION
**Pull request by brendancol**
- [x] add `constants.py` and btax related tooltips
- [x] add variable tooltips to btax output page
- [x] fix industry table css
- [ ] get addtional tooltip content for other controls
- industry
- asset
- reform
- baseline
- change from reform
- [ ] add groupings of output category

### CODE REVIEW

**Comment by brendancol** [comment]
screen shot 2016-10-21 at 3 16 25 pm

**Comment by talumbau** [comment]
@brendancol looks good! Can you add the updated `__mock_results.json` to this PR or provide to me and @PeterDSteinberg ?

**Comment by brendancol** [comment]
@talumbau @PeterDSteinberg I added two supporting files which I used as mock `table_json` for constructing the btax result table.  `__mock_btax_result.json` is the current version of table_json produced for the table, `mock_btax_result.json` is a modified version which contains `major_grouping`, `summary_nc`, and `summary_n` fields

File: .travis.yml [hunk 1]
```diff
@@ -19,8 +19,9 @@ install:
   - pip install pytest-django
   - pip install pytest-xdist
   - export SECRET_KEY="abcde%!z@cu@o29$zjifeoudt1ute&0ii33!8huud71&4#l2#817pei"
-  - git clone https://github.com/open-source-economics/B-Tax
-  - cd B-Tax && python setup.py develop && export BTAX_CUR_DIR=`pwd`/btax && cd ..
+  - git clone https://github.com/PeterDSteinberg/B-Tax
+  - cd B-Tax && git fetch --all && git checkout psteinberg/fix-groupings
```
Comment (talumbau) [inline .travis.yml]: The check out of this branch is just temporary, correct? Just pointing this out so we don't forget.

Comment (PeterDSteinberg) [inline .travis.yml]: Correct.  Just until we merge that PR in B-Tax that adds `DEFAULT_ROW_GROUPINGS` global var.

**Comment by talumbau** [comment]
@PeterDSteinberg I don't think you can get DEFAULT_ROW_GROUPINGS like you have in your PR until you merge the PR into master because the PR is set up with source branch as in your fork

**Comment by talumbau** [comment]
LGTM. merging.

**Pull request merged**